doc/ref_model/refs.bib

```diff
@@ -348,3 +348,587 @@ @misc{anuket-abbreviatons
 	title = {{Abbreviationsy of the Anuket Specifications}},
 	url = {https://cntt.readthedocs.io/en/latest/common/abbreviations.html}
 }
+
+@misc{onap-vnfrqts-requirements,
+	title = {{ONAP VNF or PNF Requirements Documentation}},
+	url = {https://docs.onap.org/projects/onap-vnfrqts-requirements/en/istanbul/}
+}
+
+@misc{owasp,
+	title = {{Open Web Application Security Project}},
+	url = {https://owasp.org/}
+}
+
+@misc{owasp-top-ten,
+	title = {{OWASP Top Ten Security Risks}},
+	url = {https://owasp.org/www-project-top-ten/}
+}
+
+@misc{owasp-Web-application-security-project,
+	title = {{Open Web Application Security Project}},
+	url = {https://owasp.org/}
+}
+
+@misc{owasp-Cheat-sheet-series,
+	title = {{OWASP Cheat Sheet Series}},
+	url = {https://github.com/OWASP/CheatSheetSeries}
+}
+
+@misc{iso-standard-83853,
+	title = {{Information technology — Redfish scalable platforms management API specification}},
+	url = {https://www.iso.org/standard/83853.html}
+}
+
+@misc{redfish-data-model-specification,
+	title = {{Redfish Data Model Specification}},
+	url = {https://www.dmtf.org/sites/default/files/standards/documents/DSP0268_2022.2.pdf}
+}
+
+@misc{redfish-interop-validator,
+	title = {{Redfish-Interop-Validator}},
+	url = {https://github.com/DMTF/Redfish-Interop-Validator}
+}
+
+@misc{redfish-data-model-specification,
+	title = {{Redfish Data Model Specification}},
+	url = {https://www.dmtf.org/sites/default/files/standards/documents/DSP0268_2023.2.pdf}
+}
+
+@misc{redfish-composition-whitepaper,
+	title = {{Redfish Composition Whitepaper}},
+	url = {https://www.dmtf.org/sites/default/files/standards/documents/DSP0268_2023.2.pdf}
+}
+
+@misc{redfish-telemetry-whitepaper,
+	title = {{Redfish Telemetry Whitepaper}},
+	url = {https://www.dmtf.org/sites/default/files/standards/documents/DSP2051_1.0.0.pdf}
+}
+
+@misc{precision-time-protocol-version-2,
+	title = {{Precision Time Protocol version 2}},
+	url = {https://standards.ieee.org/ieee/1588/6825/}
+}
+
+@misc{syncE,
+	title = {{SyncE}},
+	url = {https://www.itu.int/rec/T-REC-G.8262}
+}
+
+@misc{ITU-T G.8275.2,
```

Comment:
This is a duplicate entry. `itutg82752` refers to the same document.

Reply:
Removed @misc{ITU-T G.8275.2,
	title = {{ITU-T G.8275.2}},
	url = {https://www.itu.int/rec/T-REC-G.8275.2/en}
}